Jenkinsfile

```diff
@@ -1,20 +1,16 @@
 pipeline {
     agent any
-
-    options {
-        buildDiscarder(logRotator(numToKeepStr:'10'))
-        timeout(time: 15, unit: 'MINUTES')
-        ansiColor('xterm')
-    }
     
+    environment {
+        region = 'us-east-1'
+    }
     stages {
-        stage('Build Image') {
+        stage('Build Rhel7 Image') {
             steps {
-                script {
-                    sh "ls"
-                    
-                    sh "packer build -var-file=variables.json packer-rhel7-ami.json"
-                    
+                withCredentials([file(credentialsId: 'hailstone_secrets', variable: 'secrets')]) {
+
+                   // sh 'docker run -e REGION=$region --env-file=$secrets -i -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/setup.sh:/setup.sh -v $(pwd)/packer-rhel7-ami.json:/template.json -v $(pwd)/variables.json:/variables.json hashicorp/packer:light build -var-file=variables.json template.json'
+                    sh 'docker run -e REGION=$region --env-file=$secrets -i -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)"/:/workspace/ -w /workspace/ hashicorp/packer:light build -var-file=variables.json packer-rhel7-ami.json'
```

Comment:
@grugnog I have added quotes as required. The PR seems to be failing again with tests like previously it did.